### PR TITLE
Resolve alias account in TokenWipe transaction

### LINKED ISSUE #8165: Fails to resolve alias on TokenWipeAccount
### Description

The importer fails to properly resolve `TokenWipeAccountTransaction` with an AccountID having an alias present. It silently resolves `0.0.alias` to `0.0.0` without complaining. Since services supports alias in wipe we should as well and resolve it accurately.

### Steps to reproduce

Send a `TokenWipeAccountTransaction` with an alias accountId.

### Additional context

_No response_

### Hedera network

other

### Version

main

### Operating system

None

### LINKED ISSUE #8242: Flaky ContractResultServiceImplIntegrationTest test cases
### Description

There are two flaky test cases in `ContractResultServiceImplIntegrationTest`

```
> Task :importer:test

ContractResultServiceImplIntegrationTest > processContractActionMissingCaller() FAILED
    java.lang.AssertionError at ContractResultServiceImplIntegrationTest.java:868

ContractResultServiceImplIntegrationTest > processGasConsumedCalculationContractCall() FAILED
    org.opentest4j.AssertionFailedError at ContractResultServiceImplIntegrationTest.java:390
```

### Steps to reproduce

Run the importer test suite locally

### Additional context

_No response_

### Hedera network

other

### Version

v0.105.0-SNAPSHOT

### Operating system

None

### PR DESCRIPTION
**Pull request by xin-hedera**
**Description**:

- Resolve potential alias account in `TokenWipeTransactionHandler` and log recoverable error if it fails
- Fix flaky `ContractResultService` integration test cases

**Related issue(s)**:

Fixes #8165
Fixes #8242 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/domain/ContractResultServiceImplIntegrationTest.java [hunk 1]
```diff
@@ -839,7 +833,7 @@ private void assertContractLogs(RecordItem recordItem) {
         var contractFunctionResult = getFunctionResult(recordItem);
         var listAssert = assertThat(contractLogRepository.findAll()).hasSize(contractFunctionResult.getLogInfoCount());
         var transactionHash = getTransactionHash(recordItem);
-        Integer transactionIndex = transaction.getIndex();
+        var transactionIndex = recordItem.getTransactionIndex();
```
Comment (xin-hedera) [inline hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/domain/ContractResultServiceImplIntegrationTest.java:836]: wrong expected index causes `processContractActionMissingCaller` to fail

File: hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/domain/ContractResultServiceImplIntegrationTest.java [hunk 2]
```diff
@@ -385,15 +385,9 @@ void processGasConsumedCalculationContractCall() {
...
-        final var contractResult = contractResultRepository
-                .findById(recordItem.getConsensusTimestamp())
-                .orElse(null);
-
-        assertThat(contractResult)
-                .isNotNull()
-                .extracting(ContractResult::getGasConsumed)
-                .isNotNull()
-                .isEqualTo(22074L);
+        assertThat(contractResultRepository.findById(recordItem.getConsensusTimestamp()))
```
Comment (xin-hedera) [inline hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/domain/ContractResultServiceImplIntegrationTest.java:388]: the flakiness comes from the randomness and different gas cost of 0 / non-zero byte in contract call body's function parameters, thus using non-zero byte array as the body's function parameter eliminates the non-deterministic issue

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by jnels124 — APPROVED** [review]
LGTM

**Review by jascks — APPROVED** [review]
LGTM

**Pull request merged**